deploy/fargate_sidecar_example.json

```diff
@@ -2,6 +2,9 @@
     "executionRoleArn": "<YOUR_TASK_EXECUTION_ROLE>",
     "networkMode": "awsvpc",
     "requiresCompatibilities": ["FARGATE"],
+    "runtimePlatform": {
+      "cpuArchitecture": "X86_64"
```

Comment:
would make sense to add also the arm one commented out with some notes?

Reply:
yes but is json :sad

Reply:
https://github.com/newrelic/docs-website/pull/4873

Reply:
remeber that this is an example task template, I would expect that the original task with the service will already have this configuration in place.